### ISSUE TITLE
Select already created tags

### ISSUE THREAD

**Issue by finben** [HymnalDreamTeam/Hymns-iOS#397]
We really need a way that people can tag songs with tags they have already created instead of having to start from scratch each time.

**Comment by skywalkerdude** [comment]
@finben do you mean like a drop down or something of tags they've created already?

**Comment by skywalkerdude** [comment]
assigning to @Emyeh to explore

**Comment by finben** [comment]
@Emyeh Another consideration is the ease of access for tags. I watched my wife try out the app for the first time without saying anything just to see how she would engage with the app. I was shocked to see her frustration using tags. After she created the tag she was super confused how to access her tagged songs. She didn't find it under browse for a long time.

Would it be better to have tabs on the home screen with the first tab being browse but a second tab being tags? Or would that look bad?

**Comment by Emyeh** [comment]
I agree I was thinking about the placement of our tags. I'm not sure it makes sense to have it only in Browse. I think it needs to live within Favorites. I'll explore different variations, but my first thought is that favorites should be the home where you organize and label your hymns.

+1 on being able to select tags you've already created. It's easy to forget and create duplicate labels with different colors.

Will work on this!

**Comment by Emyeh** [comment]
Added mocks for selecting existing tags within the tag bottom sheet. All the different states should be there: 
![image](https://user-images.githubusercontent.com/26554065/88765567-8564dc00-d12b-11ea-8990-570956dcce96.png)

https://www.figma.com/file/7H5ApjzuVuJL3OP7FvE2qe/h-app?node-id=647%3A3155

Lmk if you have feedback or any questions.

**Comment by Emyeh** [comment]
Added collapse:
![image](https://user-images.githubusercontent.com/26554065/88881171-91ac7000-d1e3-11ea-8bc1-72cfda7130e4.png)

https://www.figma.com/proto/7H5ApjzuVuJL3OP7FvE2qe/h-app?node-id=1860%3A256&viewport=1682%2C418%2C0.9160079956054688&scaling=min-zoom
